src/pages/staticPages/tools/mutationMapper/MutationMapperTool.tsx

```diff
@@ -661,4 +661,36 @@ export default class MutationMapperTool extends React.Component<
         // reload the page to initialize genome nexus api instance with correct url
         getBrowserWindow().location.reload();
     }
+
+    protected grch38Info() {
```

Comment:
might as well make this a computed getter.   this will have neglibile impact, but is a best practice

Reply:
it basically will cache so React doesn't have to recompute virtual DOM ever time

Reply:
@alisman this is just a block of info and doesn't have anything to compute, I'm not sure is it still makes sense to have `computed get`